automation/taskcluster/upload_coverage_report.sh

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# If a command fails then do not proceed and fail this script too.
+set -ex
+
+# Get token for uploading to codecov and append it to codecov.yml
+python automation/taskcluster/helper/get-secret.py \
+    -s project/mobile/fenix/codecov \
```

Comment:
Sounds like this is in `project/mobile/fenix/pr` now?

Reply:
Fixed.